packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/createproject.md

```diff
@@ -1,154 +1,144 @@
 Launch Xcode and create a new iOS app project using SwiftUI for the interface and Swift for the language.
 
-> **Note:** The sample code is compatible with iOS 14 and later.
+> **Note:** The sample code uses features of SwiftUI available in iOS 14, and the async/await feature of iOS 15. Use the completion handler version of the async/await functions to add compatibility with iOS 14 and later.
 
 #### Add the UI
 
-The main view implements all of the app functionality, including initializing the Okta SDK. Note that best practice for a full app uses a singleton to manage the interaction with the SDK.
+The functionality of the app is implemented in `ContentView.swift`, the main SwiftUI view. This includes view state, view model updates, and data model. For a production app, consider partitioning the view and data models into singletons, and then sharing state using Combine, bindings, or `@Environment`.
 
-Text views display the current state and any requested information, such as the access token. The available buttons depend on the sign-in state of the user. Update the imported modules and replace the `ContentView` `struct` with the following code:
+The text views in the app display its current state and any optional information, such as the access token. The available buttons depend on the sign-in state of the user. The top-level `ZStack` contains a full-screen view that prevents the user interacting with the UI while the app is performing certain asynchronous operations, such as reading the user information. A production app may also perform asynchronous operations that do require showing a busy indicator but don't require blocking the UI.
+
+The following image shows the completed app in both the sign-in and signed-on states:
+
+<div class="full border">
+
+![Two iPhone screenshots that show the signed-out and signed-in UI.](/img/sign-users-in/redirect-authentication/ios/app-ui-ios.png)
+
+</div>
+
+<!--
+Source image(s): https://www.figma.com/file/i3huE0gEoISu2evquOq5yJ/app-ui-redirect-ios?node-id=0%3A1
+There's only one board. The group for the image is: "Side-by-side screenshots"
+-->
+
+Open `ContentView.swift` and use the following code to replace the `import` statements and the `ContentView` struct:
```

Comment:
It's more than the struct, it's also the import statements. This is important because a SwiftUI file has at least 3 parts that should be there: the framework import statements, the struct defining the view, the preview struct. With that last item there's no dynamic preview in the development environment. I didn't want to add those lines of code as they're not relevant to the task at hand.

That's why it's replacing the import statements and view struct. If you'd still like to change the wording, do you have a suggestion? Or is OK as is?

Reply:
I don't think switching the text around as suggested makes it any less obvious that both are being replaced because otherwise you appear to be omitting what to repalce the import statements with. I don't see anything about using an ampersand in the style guide, so 

> replace the `import` statements & the `ContentView` struct with the following code:

joins the two together more obviously. If that still doesn't work for you, leave as is.

Reply:
OK... this is what's in the next commit:

> Open `ContentView.swift` and replace the `import` statements and the `ContentView` struct with the following code:

Reply:
That's lovely. Thank you.